alerta/auth/oidc.py

```diff
@@ -69,6 +69,7 @@ def openid():
         'redirect_uri': request.json['redirectUri'],
         'client_id': request.json['clientId'],
         'client_secret': current_app.config['OAUTH2_CLIENT_SECRET'],
+        'scope' : 'openid profile email',
```

Comment:
Where in the spec does it suggest that scopes should be sent? This shouldn't be required.